### ISSUE TITLE
Gutter helper

### ISSUE THREAD

**Issue by walbourn** [Microsoft/UVAtlas#1]
The function [ID3DXTextureGutterHelper](https://msdn.microsoft.com/en-us/library/windows/desktop/bb174261.aspx) is not required, but can be useful for UVAtlas. It is not currently implemented as part of DirectXTex or DirectXMesh, and like UVAtlas itself combines elements of texture and mesh processing.

This could be useful functionality to consider adding to the library.

ID3DXTextureGutterHelper.ResampleTex was added in the DirectX SDK (August 2005) release.

This is required to implement the '/rt' switch for the uvatlas sample.

**Comment by xyzr0482** [comment]
Yes, I need ID3DXTextureGutterHelper.ResampleTex, when it will be added?

**Comment by walbourn** [comment]
The original UVAtlas sample switches here:

* `/rt file`  Resamples a texture using the new UVAtlas parameterization. The resampled texture is saved to a filename with "_resampled" appended.  Defaults to reading old texture parameterization from D3DDECLUSAGE_TEXCOORD[0] in original mesh. Use /rtu and /rti to override this.  

* `/rtu usage`  Specifies the vertex data usage for texture resampling. It must be one of NORMAL, POSITION, COLOR, TEXCOORD, TANGENT, or BINORMAL.  TEXCOORD  

* `/rti int`  Specifies the usage index for texture resampling.  0
